mandatory/2-function-creation.js

```diff
@@ -16,7 +18,17 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
```

Comment:
Try testing some 1 decimal place numbers e.g. 3.1, 10.2 etc with num.toFixed(2)
It maybe that you do not need to use num.toFixed(1) at all!